src/core.js

```diff
@@ -11,7 +11,11 @@ import { defined, extend, objectType, is, now } from "./core/utilities";
 import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
 
-const QUnit = {};
+var QUnit = {};
```

Comment:
Why do we need to change this to `var`? I don't see the `QUnit` local being overwritten here, am I missing something?